appveyor-nightly.yml

```diff
@@ -65,8 +65,8 @@ test_script:
 
 after_test:
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, ubuntu22]"
-  - rake "download_portable[develop]"
-  - appveyor PushArtifact "C:/projects/ospsuite-r/pksim_minimal.zip"
+#  - rake "download_portable[develop]"
```

Comment:
Hm why?

Reply:
Temporarily .
Will restore as soon as I figured out what's going on with the build foder